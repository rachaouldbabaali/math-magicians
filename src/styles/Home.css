.home-container {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: flex-start;
    height: 100%;
    padding: 40px;
  }
  
  .home-title {
    font-size: 48px;
    font-weight: bold;
    margin-bottom: 20px;
    text-align: left;
  }
  
  .home-text {
    font-size: 24px;
    line-height: 1.5;
    text-align: left;
    margin-bottom: 40px;
  }