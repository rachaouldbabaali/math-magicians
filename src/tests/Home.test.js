import renderer from 'react-test-renderer';
import Home from '../components/home';

describe('Home', () => {
  it('renders the component', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
