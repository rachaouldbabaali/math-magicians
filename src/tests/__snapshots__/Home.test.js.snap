// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the component 1`] = `
<div
  className="home-container"
>
  <h1
    className="home-title"
  >
    Welcome to our page
  </h1>
  <p
    className="home-text"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam voluptatum, voluptate, quibusdam, quia voluptas dolorum quod voluptatibus quos natus quae voluptatem? Quisquam, voluptatem voluptatum. Quisquam, voluptatem voluptatum.
  </p>
  <p
    className="home-text"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam voluptatum, voluptate, quibusdam, quia voluptas dolorum quod voluptatibus quos natus quae voluptatem? Quisquam, voluptatem voluptatum. Quisquam, voluptatem voluptatum.
  </p>
</div>
`;
